index.js

```diff
@@ -27,16 +38,31 @@ WebpackNotifierPlugin.prototype.compileMessage = function(stats) {
 
     this.lastBuildSucceeded = false;
 
-    var message;
-    if (error.module && error.module.rawRequest)
-        message = error.module.rawRequest + '\n';
+    var body, file, line, column;
 
-    if (error.error)
-        message = 'Error: ' + message + error.error.toString();
-    else if (error.warning)
-        message = 'Warning: ' + message + error.warning.toString();
+    if (error.module && error.module.rawRequest) {
+      file = error.module.userRequest;
+      body = error.module.rawRequest + '\n';
+    }
+
+    if (error.error) {
+        body = 'Error: ' + body + error.error.toString();
+        line = error.error.loc.line;
+        column = error.error.loc.column;
```

Comment:
No. Seems to be something that babel adds.

https://github.com/babel/babel/blob/9cfb8cb172cb143fb6faeef5e9e16c0e06f2ef96/packages/babel-core/src/transformation/file/index.js#L346